src/store/index.ts

```diff
@@ -36,7 +36,8 @@ export default new Vuex.Store({
     commonTooltipMessages: {
       unauthenticated: "Please log in or register to use this functionality!",
       publicData: "Public data can not be modified."
-    }
+    },
+    currentlyActiveProject: null
```

Comment:
This makes more sense to me on the projects store (`src/store/project.ts`) since that is supposed to handle global state related to projects.

Reply:
That's fair, I've only placed it there as this is where I was told it was managed in the previous platform.